tests/integration/auth.spec.coffee

```diff
@@ -109,7 +129,42 @@ describe 'SDK authentication', ->
 
 		givenLoggedInUser()
 
-		describe 'resin.auth.isLogged()', ->
+		describe 'resin.auth.isLoggedIn()', ->
```

Comment:
NIT: Since these are essentialy the exact same test, could we abstract them like this:
```coffee
describe 'when logged in', ->

	itShouldHaveFulfillingAuthMethods = ->
		/* all the describe calls here */

	describe 'using credentials'

		givenLoggedInUser()

		itShouldHaveFulfillingAuthMethods()

	describe 'using API key', ->

		givenLoggedInUserWithApiKey()

		itShouldHaveFulfillingAuthMethods()
```

Otherwise just change the pre-existing `describe 'when logged in', ->` to something like `describe 'when logged in with credentials', ->` so that we can distinguish the two easily.